app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetWashing.java

```diff
@@ -343,7 +343,7 @@ private void initializeWashComponent(Event wash) {
      */
     private void initializeButtonLogic(Wash wash, MaterialButton washButton) {
         String washButtonText = getString(R.string.wash) + SPACE + wash.getWashDescription() + EOL
-                + getString(R.string.from_date) + SPACE + wash.getDateTime() + EOL + getString(R.string.wash_duration)
+                + getString(R.string.from_date) + SPACE + wash.getDateTime().toDateString() + EOL + getString(R.string.wash_duration)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 133).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5526227)